types/bson/index.d.ts

```diff
@@ -182,6 +183,12 @@ export class Double {
      */
     constructor(value: number);
 
+    /**
+     * https://github.com/mongodb/js-bson/blob/master/lib/double.js#L17
+     */
+    value: number;
```

Comment:
Should this be `readonly`?

Reply:
Hmm, although I personally like immutable structures, looking at the actual [JS code](https://github.com/mongodb/js-bson/blob/dc9c1bb284a6c08e07f14e700bdbfb0fd57e5a13/lib/double.js#L17) I think this prop is mutable. 

Happy to add `readonly` to it though, it's probably a prop you never should write to, but I think you can write to it, so not sure how to think about it from a DefinitelyTyped perspective.

Reply:
I think its fine to leave it mutable if there are use cases for it.

Reply:
I'll merge it as-is, if it is later determined this *should* be immutable, then someone can open a PR to change that behavior.